docs/basic-features/image-optimization.md

```diff
@@ -181,15 +181,15 @@ The image component has several different [layout modes](/docs/api-reference/nex
 
 **Target the image with className, not based on DOM structure**
 
-Regardless of the layout mode used, the Image component will have a consistent DOM structure of one `<img>` tag wrapped by exactly one `<span>`. For some modes, it may also have a sibling `<span>` for spacing. These additional `<span>` elements are critical to allow the component to prevent layout shifts.
+For all of the standard layout modes, the Image component will have a consistent DOM structure of one `<img>` tag wrapped by exactly one `<span>`. For some modes, it may also have a sibling `<span>` for spacing. These additional `<span>` elements are critical to allow the component to prevent layout shifts.
```

Comment:
Its not clear what "standard" means here. Lets just update this to say that raw doesn't have a wrapper.

I think we also need to update the table of layouts to include wrappers for each `layout`.

Because this is not going to be

Reply:
I removed the "standard" verbiage and added a new column for wrapper behavior